src/components/common/Error.tsx

```diff
@@ -1,13 +1,12 @@
-import React from "react";
 import { Box, Typography, Button } from "@mui/material";
 import ErrorOutlineIcon from "@mui/icons-material/ErrorOutline";
 
-interface ErrorProps {
+interface Props {
   message: string;
   onRetry?: () => void;
```

Comment:
if you already use it render the retry function with it, if its ugly than delete this prop